### PR TITLE
[15.0][OU-ADD] coupon_criteria_multi_product: migration script

### PR DESCRIPTION
**Pull request by chienandalu**
Coming from a previous version we expect to have sale_coupon_criteria_multi_product installed despite the split

cc @Tecnativa TT39032

review @pedrobaeza @stefan-tecnativa

### CODE REVIEW

File: coupon_criteria_multi_product/migrations/15.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -9,7 +9,18 @@
 _table_renames = [("sale_coupon_criteria", "coupon_criteria")]
 
 
+def install_new_modules(env):
```
Comment (sergio-teruel) [inline coupon_criteria_multi_product/migrations/15.0.1.0.0/pre-migration.py:12]: Private method?? _install_new_modules...

Comment (chienandalu) [reply coupon_criteria_multi_product/migrations/15.0.1.0.0/pre-migration.py:12]: What for? Private methods only work at RPC level

**Review by stefan-tecnativa — APPROVED** [review]
Tested, look good to me.

**Comment by pedrobaeza** [comment]
This is not needed at all, as there's no previous renaming, so both modules will be installed on major upgrade.

**Pull request closed without merging**